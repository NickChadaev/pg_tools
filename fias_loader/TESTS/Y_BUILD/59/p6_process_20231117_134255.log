2023-11-17 13:42:55 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_59 postgres /home/n.chadaev@abrr.local/Y_BUILD/59/stage_gar_c_59.csv 2023-11-13
--------------------------------------------------------
[13:42:55] ... Пермский край, id_region: 59, Execute: /home/n.chadaev@abrr.local/Y_BUILD/59/stage_gar_c_59.csv, process_id:25345
[13:42:55] ... Start process
[13:42:55] ...  -- Version
[13:42:55] ...  -- Очистка от данных
[13:42:55] ...  -- Убираем индексы
[13:42:55] ...  -- Set UNLOGGED
[13:42:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[13:42:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[13:42:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[13:42:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[13:42:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[13:42:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[13:42:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[13:42:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[13:42:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[13:42:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[13:42:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_REESTR_OBJECTS_20231113_2b0564a8-47ff-46d5-ae37-7adde1c6777a.XML
[14:09:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_ADDR_OBJ_20231113_e5d563a8-a7a3-473a-a88a-f30ae0ecd7d0.XML
[14:10:20] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_HOUSES_20231113_57454a39-6d76-4576-ba57-fbf855c14468.XML
[14:34:46] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_STEADS_20231113_163b8f32-7219-47b6-a3df-31b6620d2ba7.XML
[14:39:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_APARTMENTS_20231113_4967bc2a-e593-4d41-9304-7e31519d666a.XML
[14:51:21] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_ROOMS_20231113_cc96498a-dabd-4987-a37b-3158aacf5c48.XML
[14:52:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_CARPLACES_20231113_0e45b3e5-2e5c-4ba6-bbd4-3b461fbddacc.XML
[14:52:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_ADM_HIERARCHY_20231113_11dedaaf-e77d-4aa7-9a9d-321a44d8bfe1.XML
[15:22:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_MUN_HIERARCHY_20231113_66fab5c1-0e9e-4c79-9a9f-88d33a57bddd.XML
[16:02:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_CHANGE_HISTORY_20231113_0cdbd7f3-9da1-480b-b476-c31f480c2d5c.XML
[17:40:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_ADDR_OBJ_DIVISION_20231113_7878c86a-3df6-4dcd-9520-6f2abf66c208.XML
[17:40:28] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_ADDR_OBJ_PARAMS_20231113_b59c93c6-82d1-4cdb-916a-4d468c2c676b.XML
[17:48:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_STEADS_PARAMS_20231113_5560473c-fe33-48c0-a742-83dd76c19b66.XML
[18:30:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_HOUSES_PARAMS_20231113_90e055a4-0d54-440b-81a9-382b7068af8d.XML
[20:30:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_APARTMENTS_PARAMS_20231113_f7527554-8a46-4a21-9b1e-59f7612de97b.XML
[20:53:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_ROOMS_PARAMS_20231113_10c08844-d2f2-4c11-8571-e389e3a3112f.XML
[20:53:46] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_CARPLACES_PARAMS_20231113_ae9ac7b1-e726-407c-933c-f1a6cd783f9d.XML
[20:53:46] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/59/AS_NORMATIVE_DOCS_20231113_9838ebf6-6431-4d2c-bdd9-af40f893c15c.XML
[20:57:26] ...  -- Data finish
[20:57:26] ...  -- Set LOGGED
[20:58:25] ... Stop process
--------------------------------------------------------
