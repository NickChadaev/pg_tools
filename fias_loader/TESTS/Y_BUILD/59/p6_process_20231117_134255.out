 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 2159596
gar_fias.as_addr_obj: 55067
gar_fias.as_houses: 1818234
gar_fias.as_steads: 336512
gar_fias.as_apartments: 984961
gar_fias.as_rooms: 52439
gar_fias.as_carplaces: 6
gar_fias.as_adm_hierarchy: 2425399
gar_fias.as_mun_hierarchy: 3080302
gar_fias.as_change_history: 7628725
gar_fias.as_addr_obj_division: 137
gar_fias.as_addr_obj_params: 665488
gar_fias.as_steads_params: 3369203
gar_fias.as_houses_params: 9876040
gar_fias.as_apartments_params: 1851176
gar_fias.as_rooms_params: 52181
gar_fias.as_carplaces_params: 6
gar_fias.as_normative_docs: 311201
